soc/wch/ch32v00x/Kconfig

```diff
@@ -0,0 +1,9 @@
+config SOC_CH32V003
```

Comment:
missing copyright/license header on files

Reply:
This problem should be fixed now (across the PR) to a good extent.